documentation/components/tutorial-component-basic-features.asciidoc

```diff
@@ -23,16 +25,23 @@ component.setId("my-component");
 
 == Element
 
-Every component has a mapped `Element` that can be used to access low level functionality, by using `component.getElement()`.
-For more details, see <<../element-api/tutorial-properties-attributes#,Element Properties and Attributes>>.
+Every component has a mapped `Element`.
```

Comment:
```suggestion
Every component is assosiated to a root `Element`.
```

Reply:
I don't know how it is "mapped". Each Component has an Element (that may may include other elements). I'd though put it like this.